blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java

```diff
@@ -61,13 +64,32 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
 
         Tracker.getInstance().track(request);
 
+        requestCount.inc();
```

Comment:
If an exception is thrown between this call to `inc()` and the subsequent `try` on line 90, or if the methods `return`s during that span, then the corresponding call to `dec()` on line 139 won't get called, and `timerContext.stop()` won't get called on line 140. Perhaps a second `try/finally` pair should be used to make sure those methods get called?